src/main/java/Duke.java

```diff
@@ -1,10 +1,141 @@
+    private static final String LOGO = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+                + "|____/ \\__,_|_|\\_\\___|\n\n";
+    private static final String MESSAGE_GREETING = " Hello! I'm Duke" + "\n What can I do for you?";
+    private static final String MESSAGE_ENDING = " Bye. Hope to see you again soon!";
+    private static final String MESSAGE_LIST = " Here are the tasks in your list:";
+    private static final String MESSAGE_ADDTASK = " Got it. I've added this task:";
+    private static final String MESSAGE_REMAININGTASK = " Now you have %d tasks in the list.";
+    private static final String MESSAGE_DONETASK = "Nice! I've marked this task as done:";
+
+    /**
+     * Prints out the response to the user with line boundaries surrounding it
+     *
+     * @param respones message that is shown to the user
+     *
+     */
+    public static void printResponse(String response) {
+        System.out.println(LINE_SEPARATOR);
+        System.out.println(response);
+        System.out.println(LINE_SEPARATOR);
+    }
+
+    /**
+     * Prints out the full list of tasks with line boundaries surrounding it
+     *
+     * @param tasks list of tasks to be printed out
+     * @param response message that is shown to the user
+     *
+     */
+    public static void printResponse(ArrayList<Task> tasks, String response) {
+        System.out.println(LINE_SEPARATOR);
+        System.out.println(response);
+        for (int i = 0; i < tasks.size(); i ++) {
+            System.out.println(" " + (i + 1) + ". " + tasks.get(i).toString());
+        }
+        System.out.println(LINE_SEPARATOR);
+    }
+
+    /**
+     * Prints out either the newly added task or completed task
+     *
+     * @param tasks to access specific task in the Arraylist
+     * @param response message that is shown to the user
+     * @param taskNo either the newly added task no. or the completed task no.
+     *
+     */
+    public static void printResponse(ArrayList<Task> tasks, String response, int taskNo) {
+        System.out.println(LINE_SEPARATOR);
+        System.out.println(response);
+        if (response.equals(MESSAGE_ADDTASK)) {
+            System.out.println("   " + tasks.get(taskNo - 1).toString());
+            System.out.println(String.format(MESSAGE_REMAININGTASK, taskNo));
+        } else {
+            System.out.println("   " + tasks.get(taskNo - 1).toString());
+        }
+        System.out.println(LINE_SEPARATOR);
+    }
+
+    /**
+     * Separates the userInput to description and time string
+     *
+     * @param description userInput to be separated
+     * @return inputs description and time is stored in the string array
+     *
+     */
+    public static String[] separateDescriptionAndTime(String description) {
```

Comment:
It's quite good to introduce a method for the separation operation so that dealing with input is divided into small tasks.